code/modules/vehicles/mecha/_mecha.dm

```diff
@@ -547,33 +538,29 @@
 		return
 	if(internal_damage & MECHA_INT_CONTROL_LOST)
 		target = pick(view(3,target))
-		if(!target)
-			return
```

Comment:
?

Reply:
Please clarify what you mean here

Reply:
I was confused about why the sanity check was removed. On closer inspection it was shitcode